src/Umbraco.Web/Models/ContentEditing/ContentTypeCompositionDisplay.cs

```diff
@@ -44,6 +44,10 @@ public ContentTypeCompositionDisplay()
         [DataMember(Name = "compositeContentTypes")]
         public IEnumerable<string> CompositeContentTypes { get; set; }
         
+        //Locked compositions
+        [DataMember(Name = "lockedCompositeContentTypes")]
+        public IEnumerable<string> LockedCompositeContentTypes { get; set; }

```

Comment:
I needed that one for ensuring you cannot deselect the composition (which is really not a composition, but an inherited model) in the UI. But let's see once we get everything merged.